app-crypt/nitrocli/nitrocli-0.1.3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CRATES="
+cc-1.0.25
+hid-0.4.1
+hidapi-sys-0.1.4
+libc-0.2.45
+nitrocli-0.1.3
+pkg-config-0.3.14
+"
+
+inherit cargo
+
+DESCRIPTION="A command line tool for interacting with the Nitrokey Storage."
+HOMEPAGE="https://github.com/d-e-s-o/nitrocli"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+RESTRICT="mirror"
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+# We require gnupg for /usr/bin/gpg-connect-agent.
+RDEPEND="
+	app-crypt/gnupg
+	dev-libs/hidapi
+"
+DEPEND="
+	>=dev-lang/rust-1.31.0
+	${RDEPEND}
+"
+
+src_install() {
+	cargo install -j $(makeopts_jobs) --path=. --root="${D}/usr" $(usex debug --debug "") \
```

Comment:
Isn't this what default cargo_src_install calls?

Reply:
you can omit calling `cargo_src_install` altogether and just install binary with dobin
it's located in `target/release/nitrocli`
it's way simpler than asking cargo to install a single binary file.